<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>签到程序</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tabSign">
       <attribute name="title">
        <string>签到</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="signEdit">
          <property name="font">
           <font>
            <family>PingFang SC</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="labelName">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditEname">
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>姓名：</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton">
             <property name="toolTip">
              <string>点击首字母查询</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelGender">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxEgender">
             <property name="editable">
              <bool>true</bool>
             </property>
             <item>
              <property name="text">
               <string>女</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>男</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelPhone">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditEphone">
             <property name="inputMethodHints">
              <set>Qt::ImhDigitsOnly</set>
             </property>
             <property name="inputMask">
              <string>0000 0000 000</string>
             </property>
             <property name="placeholderText">
              <string>手机号：</string>
             </property>
             <property name="cursorMoveStyle">
              <enum>Qt::VisualMoveStyle</enum>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelBirth">
             <property name="text">
              <string>生日：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dateEditEBirth">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="displayFormat">
              <string>yyyy-MM-dd</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelDate">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dateEditEcurrent">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::UpDownArrows</enum>
             </property>
             <property name="currentSection">
              <enum>QDateTimeEdit::YearSection</enum>
             </property>
             <property name="displayFormat">
              <string>yyyy-MM-dd</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonEsave">
             <property name="font">
              <font>
               <family>STLiti</family>
              </font>
             </property>
             <property name="text">
              <string>保存</string>
             </property>
             <property name="shortcut">
              <string>Return</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tableViewDict">
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="autoScroll">
           <bool>false</bool>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="dragDropOverwriteMode">
           <bool>false</bool>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::ActionMask</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tableViewSigns">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabQuery">
       <attribute name="title">
        <string>查询</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="queryGroup">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="labelQName">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditQName">
             <property name="placeholderText">
              <string>姓名：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelQGender">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxQGender">
             <item>
              <property name="text">
               <string>女</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>男</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelQDay">
             <property name="text">
              <string>(生日) 日:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxDay">
             <property name="minimumSize">
              <size>
               <width>48</width>
               <height>24</height>
              </size>
             </property>
             <property name="showGroupSeparator" stdset="0">
              <bool>false</bool>
             </property>
             <property name="displayIntegerBase">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelQMonth">
             <property name="text">
              <string>(生日) 月：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxMonth"/>
           </item>
           <item>
            <widget class="QLabel" name="labelQPhone">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditQPhone">
             <property name="placeholderText">
              <string>电话：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonQ">
             <property name="text">
              <string>query</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTableView" name="tableViewQDict">
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
          </widget>
          <widget class="QTableView" name="tableView">
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabData">
       <attribute name="title">
        <string>导数</string>
       </attribute>
       <widget class="QGroupBox" name="groupBoxDataExport">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>162</width>
          <height>145</height>
         </rect>
        </property>
        <property name="title">
         <string>==&gt; 按日期导出签到数据: &lt;==</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QDateEdit" name="dateEditDataExportStart">
           <property name="displayFormat">
            <string>yyyy-MM-dd</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
           <property name="date">
            <date>
             <year>2016</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEditDataExportEnd">
           <property name="displayFormat">
            <string>yyyy-MM-dd</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
           <property name="date">
            <date>
             <year>2016</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonDataExport">
           <property name="text">
            <string>导出数据</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>180</y>
          <width>181</width>
          <height>98</height>
         </rect>
        </property>
        <property name="title">
         <string>==&gt; 拖拽文件到下面松开：&lt;==</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLineEdit" name="lineEditDataImport">
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonDataImport">
           <property name="text">
            <string>导入数据</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>笔记</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QDateEdit" name="dateEditNote">
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="accelerated">
             <bool>false</bool>
            </property>
            <property name="currentSection">
             <enum>QDateTimeEdit::YearSection</enum>
            </property>
            <property name="displayFormat">
             <string>yyyy-MM-dd</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
            <property name="date">
             <date>
              <year>2016</year>
              <month>1</month>
              <day>1</day>
             </date>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>当前记录的是：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditNoteName">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>手机号为：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditNotePhone">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonSaveNote">
            <property name="statusTip">
             <string>ctrl + s</string>
            </property>
            <property name="text">
             <string>保存记录</string>
            </property>
            <property name="shortcut">
             <string>Meta+S</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonExportCurrentNote">
            <property name="text">
             <string>笔记导出</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTableView" name="tableViewNoteHuman">
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="showGrid">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QTableView" name="tableViewNoteList">
           <property name="showDropIndicator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="showGrid">
            <bool>false</bool>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEditNote"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 230, 230);</string>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBarQuery">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDict"/>
   <addaction name="actionList"/>
   <addaction name="actionCurrentDateRows"/>
   <addaction name="actionExport"/>
   <addaction name="actionUpdatePerson"/>
   <addaction name="actionFirstLetter"/>
  </widget>
  <action name="actionDict">
   <property name="text">
    <string>dict</string>
   </property>
  </action>
  <action name="actionList">
   <property name="text">
    <string>list</string>
   </property>
  </action>
  <action name="actionCurrentDateRows">
   <property name="text">
    <string>currentDateRows</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>export</string>
   </property>
   <property name="toolTip">
    <string>备份数据库到文档路径</string>
   </property>
  </action>
  <action name="actionUpdatePerson">
   <property name="text">
    <string>updatePersonByName</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;用来当前记录的人名的所有信息， 在编辑标签的页面中， 更改除人名以外的所有信息，点击此按钮后会按当前输入更新所有此姓名的信息。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionFirstLetter">
   <property name="text">
    <string>firstLetter</string>
   </property>
   <property name="shortcut">
    <string>Meta+Return</string>
   </property>
  </action>
  <action name="actionDeleteRow">
   <property name="text">
    <string>删除本行信息</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditEname</tabstop>
  <tabstop>comboBoxEgender</tabstop>
  <tabstop>lineEditEphone</tabstop>
  <tabstop>dateEditEBirth</tabstop>
  <tabstop>dateEditEcurrent</tabstop>
  <tabstop>pushButtonEsave</tabstop>
  <tabstop>tableViewDict</tabstop>
  <tabstop>tableViewSigns</tabstop>
  <tabstop>tableViewQDict</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
